VCFv4.3.tex

```diff
@@ -513,28 +529,31 @@ \section{INFO keys used for structural variants}
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants, END is the corresponding best estimate.
 
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
 \end{verbatim}
 \normalsize
-This key can be derived from the REF/ALT fields but is useful for filtering. The reserved values must be used for the types listed below:
+SVTYPE is a primitive basic data type and must be specified for all structural variants. Its value can usually be deduced from the REF and ALT fields, however despite this redundancy it is quite useful for filtering. For each VCF record, SVTYPE must be assigned exactly one of the following values:
+
 \begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
+\item DEL:  Deletion relative to the reference
+\item INS:  Insertion relative to the reference
+\item DUP:  Region of elevated copy number relative to the reference
+\item INV:  Inversion of reference sequence
+\item CNV:  Copy number variable region (multiallelic)
+\item BND:  Breakend
 \end{itemize}
 
+
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
 \end{verbatim}
 \normalsize
-One value for each ALT allele. Longer ALT alleles (e.g. insertions) have positive values, shorter ALT alleles (e.g. deletions) have negative values.
+SVLEN indicates the relative size of the ALT allele compared to the REF allele. If ALT is longer than REF (e.g. insertions), SVLEN must be positive; if ALT is shorter than REF (e.g. deletions), SVLEN must be negative. One SVLEN value for each ALT allele.
 
```

Comment:
Perfect inversions will have SVLEN=0. The size of the inversion can be determined by the POS field and the END attribute.

Reply:
Added.